be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -106,9 +106,9 @@ StatusOr<vectorized::ChunkPtr> ScanOperator::pull_chunk(RuntimeState* state) {
     }
 
     auto&& chunk = _chunk_source->get_next_chunk_from_buffer();
-    // If buffer size is smaller than half of batch_size,
+    // If buffer size is smaller than half of buffer_size,
```

Comment:
buffer size is smaller than half of buffer_size ?

Reply:
> buffer size is smaller than half of buffer_size ?

I will rename this